core/src/test/java/org/hibernate/ogm/backendtck/associations/collection/manytomany/ManyToManyExtraTest.java

```diff
@@ -21,7 +21,7 @@
  * @author Emmanuel Bernard &lt;emmanuel@hibernate.org&gt;
  */
 @SkipByGridDialect(
-		value = { GridDialectType.CASSANDRA, GridDialectType.INFINISPAN_REMOTE },
+		value = { GridDialectType.INFINISPAN_REMOTE },
```

Comment:
~~This bugs me a little. We are losing information and I would have expected the TCK to be used for the contrib dialects. It looks like you're reintroducing the ability to do that in a further commit. What's your strategy exactly for that?~~

We agree that you have moved this information in the skipTests file of each contrib module?